models/mongodb/rank.js

```diff
@@ -6,6 +6,8 @@ var Schema = mongoose.Schema
 var Rank = new Schema({
     candidate: Number,
```

Comment:
Lacks indexes (epoch, candidate)

Reply:
Because of Rank table is in v1.4.0, so I will prepare another PR to update the table